test/helpers/components/dynakube/dynakube.go

```diff
@@ -25,7 +25,7 @@ import (
 
 const (
 	defaultName      = "dynakube"
-	defaultNamespace = "dynatrace"
+	DefaultNamespace = "dynatrace"
```

Comment:
this was private so it forces you to relay on the created `dynakube` object and not hardcoded consts from random places of the code

if we need this (duo to the make target always using this) then it should be in a different package, as this is not a `DynaKube` specific configuration